### PR TITLE
added toast

### PR DESCRIPTION
**Pull request by MiryalaAkshitha**

### CODE REVIEW

**Review by vivekgudipudi — APPROVED** [review]

File: components/toast/toast.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                </code>
+            </pre>
+        </div>
+
+
+    </div>
+    
+
+
+
```
Comment (smkmallik) [inline components/toast/toast.html:60]: minor: remove extra spaces

File: components/toast/toast.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+.toast-icon{
+    position: absolute;
+    right: 8px;
+    top: 14px;
+    cursor: pointer;
+}
+
+.toast-text{
+    padding-right:1rem;
+}
```
Comment (smkmallik) [inline components/toast/toast.css:18]: Changes seems good.

File: components/toast/toast.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+            </div>
+            <pre>
+                <code class="html">
+    &lt;div class="toast-container"&gt;
+        &lt;div class="toast-text"&gt;Internet unavailable. Please try again.&lt;/div&gt;
+        &lt;i class="far fa-times toast-icon"&gt;&lt;/i&gt;
+    &lt;/div&gt;
+                </code>
+            </pre>
+        </div>
```
Comment (smkmallik) [inline components/toast/toast.html:53]: HTML Structure seems good as well

**Review by smkmallik — APPROVED** [review]
Changes seems okay, you can further do the responsive part when you further work on this. You can merge the changes now.

**Pull request merged**